writecrow/Crow-Contributors
# Added Mark Fullmer to the list of contributors (Resolves #3)

This pull request adds Mark Fullmer to the list of contributors on the README.

## Testing Steps
0. View the README file on this branch.
1. Verify that "Mark Fullmer" is listed, with accurate title.
